### ISSUE TITLE
Mysterious "Unexpected parsing error"

### ISSUE THREAD

**Issue by kashizui** [codalab/codalab-worksheets#307]
Getting `Unexpected parsing error` on the following line:

`[run accuracy -- hybrid-0:,up-0:,down-0:,:abstract-0,:abstract-5,:abstract-10,:abstract-15,:abstract-20,:decode-0,:decode-5,:decode-10,:decode-15,:decode-20,:get_speech_accuracy.sh : bash get_speech_accuracy.sh]{0x9997581ad6e142788f3f255208fc00d7}`

https://worksheets.codalab.org/worksheets/0x8967960a7c644492974871ee60198401/

**Comment by kashizui** [comment]
Found traceback, working on solution:

```
Traceback (most recent call last):
  File "/Users/sckoo/Developer/codalab-cli/codalab/lib/worksheet_util.py", line 820, in interpret_items
    flush_bundles()
  File "/Users/sckoo/Developer/codalab-cli/codalab/lib/worksheet_util.py", line 759, in flush_bundles
    for (name, genpath, post) in schema
  File "/Users/sckoo/Developer/codalab-cli/codalab/lib/worksheet_util.py", line 759, in <dictcomp>
    for (name, genpath, post) in schema
  File "/Users/sckoo/Developer/codalab-cli/codalab/lib/worksheet_util.py", line 397, in interpret_genpath
    key, value = friendly_render_dep(dep)
  File "/Users/sckoo/Developer/codalab-cli/codalab/lib/worksheet_util.py", line 383, in friendly_render_dep
    dep['parent_uuid'][0:4] + '}'
TypeError: unsupported operand type(s) for +: 'NoneType' and 'str'
```

**Comment by kashizui** [comment]
Fixed by #320
